src/types/store/index.ts

```diff
@@ -1,7 +1,12 @@
 import { ActionCreator, AnyAction } from 'redux';
+import { Epic } from 'redux-observable';
+
+import { RootState } from 'store';
+import { RootAction } from 'store/actions';
 
 export interface IAppAction<T> extends AnyAction {
   payload?: T;
 }
 
 export type AppActionCreator<T = undefined> = ActionCreator<IAppAction<T>>;
+export type AppEpic = Epic<RootAction, RootAction, RootState, unknown>;
```

Comment:
You may leave 2 generic options for app epic to be able to specify them for each epic. And add RootAction as default value, like: `AppEpic<TInAction = RootAction, TOutAction = RootAction> = Epic<TInAction, TOutAction, RootState, unknown>;`